addons/mass_mailing/models/mail_mail.py

```diff
@@ -57,7 +57,7 @@ def _prepare_outgoing_list(self, recipients_follower_status=None):
         view links by email-specific links. Also add headers to allow
         unsubscribe from email managers. """
         email_list = super()._prepare_outgoing_list(recipients_follower_status)
-        if not self.res_id or not self.mailing_id:
+        if (not self.res_id and self.res_id != 0) or not self.mailing_id:
```

Comment:
I don't get it, won't it always be 0 if Falsy? Or if it can be False somehow doesn't it make sense to check "is False" instead?